src/components/InteractiveTutorial/TutorialFooter.jsx

```diff
@@ -1,54 +1,42 @@
 import React from 'react';
-import { Box, Button, Divider, Grid } from '@mui/material';
-import { useNavigate, useParams } from 'react-router-dom';
-import tutorialSubPages from './TutorialSubpages';
-import { ArrowBack, ArrowForward } from '@mui/icons-material';
+import PropTypes from 'prop-types';
+import { Box, Button } from '@mui/material';
+import { Link } from 'react-router-dom';
+import { getTutorialSubPages } from './TutorialSubpages';
+import { useClient } from '../../context/client-context';
 
-export const TutorialFooter = () => {
```

Comment:
This seems like a log of changes, I propose to hide the whole tutorial section instead, and don't rewrtie internals